src/main/resources/doc/de/html/guide/subcirc/sub-appear.html

```diff
@@ -17,7 +17,7 @@ <h1>
         Auswählen des Erscheinungsbildes eines Subcircuits
       </h1>
       <p>
-        Es gibt vier Modelle für die grafische Darstellung der Schaltung (Logisim-Entwicklung,Logisim-holycross,Klassisches-Logisim,Benutzerdefiniert). Die Version der Präsentation ist in der Eigenschaft Circuit Appearance auswählbar. 
+        Es gibt vier Modelle für die grafische Darstellung der Schaltung (Logisim-evolution,Logisim-holycross,Klassisches-Logisim,Benutzerdefiniert). Die Version der Präsentation ist in der Eigenschaft Circuit Appearance auswählbar. 
```

Comment:
May I also request adding single space after each `,` in `(Logisim-evolution,Logisim-holycross,Klassisches-Logisim,Benutzerdefiniert)` line?